R/translate-sql.r

```diff
@@ -36,6 +36,7 @@
 #' @param window Use `FALSE` to suppress generation of the `OVER`
 #'   statement used for window functions. This is necessary when generating
 #'   SQL for a grouped summary.
+#' @param context Use to carry information for special translation cases.  Expects a list.
```

Comment:
For example, MS SQL needs a different conversion for `is.na()` in `WHERE` vs. `SELECT` clauses.